pkg/app/piped/platformprovider/ecs/client.go

```diff
@@ -252,7 +252,8 @@ func (c *client) WaitServiceStable(ctx context.Context, service types.Service) e
 		Cluster:  service.ClusterArn,
 		Services: []string{*service.ServiceArn},
 	}
-
+	// TODO: Wait until a new task is started instead of sleeping.
```

Comment:
```suggestion
	// Wait before first checking the service state due to the logic checking service
	// stable currently is based on `pendingCount`, which could always be `0` when
	// the service deployment has started running.
	// TODO: Wait until a new task is started instead of sleeping.
```
nits, add a note to clarify why we have to use sleep here.